src/rtcSessionHelpers.ts

```diff
@@ -149,15 +145,23 @@ const widgetPostHangupProcedure = async (
   // calling leaveRTCSession.
   // We need to wait because this makes the client hosting this widget killing the IFrame.
   await widget.api.transport.send(ElementWidgetActions.HangupCall, {});
+  // On a normal user hangup we can shut down and close the widget. But if an
+  // error occurs we should keep the widget open until the user reads it.
+  if (cause === "user") {
+    await widget.api.transport.send(ElementWidgetActions.Close, {});
```

Comment:
Can we add a way to also emit this with a button on the error screen?
This is unnecessary for EW since we have the close button there. But on EX this is a better UX since we then can close the widget by pressing sth like "Ok" or "Close" inside the IFrame.

(for now this screen would of course not be visible but once we update the rust sdk (if we go this route where a rust sdk udpate is required) we dont introduce the isse that the user cannot exit the error screen except when using the os activity manager)

Reply:
I would love to do that, yeah, but in a separate PR.

Reply:
Sounds good. I think we should try to mitigate the following situation:
 - Not being able to close the widget with a button on an error screen
 - EX updated to not close on `hangup` anymore and waiting for the close action.
 
 Then the use might feel stuck on an error screen until they close the activity.